nimble/controller/include/controller/ble_ll_conn.h

```diff
@@ -58,6 +58,8 @@ extern "C" {
 /* Definition for RSSI when the RSSI is unknown */
 #define BLE_LL_CONN_UNKNOWN_RSSI        (127)
 
+#define BLE_LL_CONN_ISO_HANDLE_OFFSET   (0x0100)
```

Comment:
I'd name this BLE_LL_CONN_HANDLE_ISO_OFFSET (so that is it similar to BLE_HS_CONN_HANDLE_*)